src/api/operator/types.ts

```diff
@@ -38,6 +40,28 @@ export type RawOrder = {
   signature: string
 }
 
+/**
+ * Enriched Order type.
+ * Applies some transformations on the raw api data.
+ * Some fields are kept as is.
+ */
+export type Order = Pick<RawOrder, 'owner' | 'uid' | 'appData' | 'kind' | 'partiallyFillable' | 'signature'> & {
+  creationDate: Date
+  expirationDate: Date
+  buyTokenAddress: string
+  sellTokenAddress: string
```

Comment:
if we enrich this, should we also enrich the token? not sure, just thinking...

Reply:
I though about this.
The reasons for keeping it out of the `Order` are:
1. Token details are not part of the raw order, thus will have to be added later
2. Avoid nested objects so React is better at identifying obj updates/component redraw

For `1`, it's easy to address, since I'm fetching the token details in the same hook
For `2`, haven't really measured the performance impact. Was more of something I'm aware but don't fully know how it works.

None are strong reasons though. Will update.

Reply:
It was ok too. I didn't have strong opinion on this one either. I see reasons to do it and not do it